HipercowApi/Tools/Utils.cs

```diff
@@ -7,20 +7,26 @@ namespace HipercowApi.Tools
     using Microsoft.Hpc.Scheduler.Properties;
 
     /// <summary>
-    /// Miscellaneous support helper functions to make the code more readable elsewhere.
+    /// Miscellaneous support helper functions to make the code more
+    /// readable elsewhere.
     /// </summary>
-    public class Utils : IUtils
+    public class Utils
     {
         /// <summary>
-        /// Wrapper around Microsoft.Hpc.Scheduler.Properties; return a string if
-        /// possible, otherwise return an empty string, rather than null. In reality,
-        /// a non-null StoreProperty can't have a null value or ToString - but I
-        /// think it is not declared that way - hence code coverage disabled for
-        /// this one-liner, as otherwise I get a partial line coverage that I can't
-        /// complete.
+        /// Wrapper around Microsoft.Hpc.Scheduler.Properties; return a string
+        /// if possible, otherwise return an empty string, rather than null.
+        /// In reality, a non-null StoreProperty can't have a null value or
+        /// ToString - but I think it is not declared that way - hence code coverage
+        /// disabled for this one-liner, as otherwise I get a partial line coverage
+        /// that I can't complete.
         /// </summary>
-        /// <param name="sp">A Microsoft.Hpc.Scheduler.Properties.StoreProperty, returned when querying the cluster.</param>
-        /// <returns>A string which either has a value or is empty, but is not null.</returns>
+        /// <param name="sp">
+        /// A Microsoft.Hpc.Scheduler.Properties.StoreProperty, returned when
+        /// querying the cluster.
+        /// </param>
+        /// <returns>
+        /// A string which either has a value or is empty, but is not null.
+        /// </returns>
         [ExcludeFromCodeCoverage]
```

Comment:
Should be able to unexclude this?

Reply:
OK - I think I know something now that I didn't when I wrote this line! The issue is that `StoreProperty.Value` is not nullable, but .`ToString()` returns a nullable `string?` - which I caught and turned into an empty, even though it shouldn't be possible to generate... and indeed it isn't, so the second half of my line was not reachable, hence always only partial code coverage.

I've changed to `return sp.Value.ToString()!;` which I think will be safe since Value really never is null.